lib/ffi.dart

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library ffi;
```

Comment:
You can consider either prefixing the library name with the package name (I believe they live in a global namespace) or leaving the library declaration away entirely.

Reply:
Removed.